### ISSUE TITLE
Bundle goon driver with porcelain.

### ISSUE THREAD

**Issue by Litterfeldt** [alco/porcelain#22]
Why not?

**Comment by alco** [comment]
It wouldn't be very practical, look at the number of versions of goon that are available https://github.com/alco/goon/releases/tag/v1.1.1

**Comment by Litterfeldt** [comment]
Sure there are a few, not all of them have to be covered.

It would be mighty convenient to have out-of-the-box mac and linux 64 bit support as that is what most of us are running.

Wrote a tiny script using `uname -a` to bootstrap goon but it didn't feel to far-fetched to have bundled versions.

**Comment by aphillipo** [comment]
Yes, I find it a little strange the goon needs to be another dependency at all - and written in go too!  Is go just better at managing OS processes in some way?

**Comment by electricshaman** [comment]
Bundling the binary would also be problematic for cross compiling.
